addons/l10n_in_edi/models/account_move.py

```diff
@@ -61,3 +62,14 @@ def _l10n_in_edi_is_managing_invoice_negative_lines_allowed(self):
         """
         param_name = 'l10n_in_edi.manage_invoice_negative_lines'
         return bool(self.env['ir.config_parameter'].sudo().get_param(param_name))
+
+    @api.depends('move_type', 'line_ids.tax_tag_ids', 'amount_residual', 'partner_id.l10n_in_pan')
+    def _compute_display_null_pan_warning(self):
+        tds_tcs_tax_tags = self.env.ref("l10n_in.tax_tag_base_tcs") | self.env.ref("l10n_in.tax_tag_base_tds")
+        for record in self:
```

Comment:
can you please filter out moves that do not belong to an Indian company? Ideally, the addition of a field in a localization module shouldn't impede the behavior for other companies. So you can assign in batch for all records of non-Indian companies

Reply:
Hello @qdp-odoo ,
Thanks for your review and I made the suggested changes. Can you please check this out?